docs/user/metal/install_ipi.md

```diff
@@ -195,3 +195,80 @@ network configuration, but fully supporting alternative subnets for the
 provisioning network is incomplete.
 
 https://github.com/openshift/installer/issues/2091
+
+## Troubleshooting
+
+General troubleshooting for OpenShift installations can be found
+[here](/docs/user/troubleshooting.md).
+
+### Bootstrap
+
+The bootstrap host is currently a libvirt VM running on the same host as
+the installer. This bootstrap host runs the Ironic services needed to
```

Comment:
We should probably be consistent with the naming, e.g "bootstrap VM" as folks sometimes get confused wrt the VM vs the provisioning host?

We have a mixture here of "bootstrap host" and "bootstrap node" - perhaps we should just be explicit and say at the start we'll refer to that as the "Bootstrap VM" and use that naming everywhere in the docs?